framework_lib/src/chromium_ec/mod.rs

```diff
@@ -236,7 +236,8 @@ impl CrosEc {
         ))
     }
 
-    /// Get current status of Framework Laptop's privacy switches (microphone and camera)
+    /// Get current status of Framework Laptop's microphone and camera privacy switches
+    /// [true = device enabled/connected, false = device disabled]
```

Comment:
This is good. I think it's important to clarify whether the "privacy switch" or the "device" is disabled/enabled.
Because the user could also see it as "privacy enabled" meaning "device disabled"